src/stream_ml/pytorch/builtin/_skewnormal.py

```diff
@@ -124,7 +126,7 @@ def log_truncation_term(
     """Log of integral from a to b of skew-normal."""
     erfa = xp.erf(skew * (ab[0] - loc) / sigma / _sqrt2)  # type: ignore[attr-defined]
     erfb = xp.erf(skew * (ab[1] - loc) / sigma / _sqrt2)  # type: ignore[attr-defined]
-    return xp.log(erfb - erfa) + xp.log(erfb + erfa + 2) - xp.log(4)
+    return xp.log(erfb - erfa) + xp.log(erfb + erfa + 2) - xp.log(Tensor(4.0))
```

Comment:
It's probably easier to do

```suggestion
    return xp.log(erfb - erfa) + xp.log(erfb + erfa + 2) - math.log(4.0)
```

Reply:
Agreed.